addons/invert-stage/addon.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "Invert Stage",
+  "description": "Inverts the stage. Useful for testing contrast and people with colorblindness.",
+  "tags": ["community"],

```

This is an editor addon, so it should have matching tags.

```suggestion
  "tags": ["editor", "projectPlayer"],
```